.changeset/thirty-nails-shake.md

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
I think we can just have the "main" changelog entry for this?

Reply:
@angrykoala thoughts?

Reply:
Fine by me